pkgs/by-name/fo/foxglove-studio/package.nix

```
@@ -0,0 +1,110 @@
+{
+  alsa-lib,
+  at-spi2-atk,
+  autoPatchelfHook,
+  cairo,
+  cups,
+  dbus,
+  dpkg,
+  eudev,
+  expat,
+  fetchurl,
+  glib,
+  glibc,
+  gtk3,
+  lib,
+  libdrm,
+  libgcc,
+  libxkbcommon,
+  makeWrapper,
+  mesa,
+  nspr,
+  nss,
+  pango,
+  stdenv,
+  systemd,
+  via,
+  xorg,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  name = "foxglove-studio";
```

```suggestion
  pname = "foxglove-studio";
```
name is constructed based on pname and version